RedHatInsights/insights-inventory-frontend
# fix(InsightsDisconnected): update the disconnected systems verification logic

Fixes: https://issues.redhat.com/browse/ESSNTL-2702

To test:

1. visit the inventory app
2. click on the filters dropdown
3. click on the 'Data collector' filter and select the 'insights-client not connected' option
4. verify that all hosts in the list have a disconnected icon next to the last seen date
5. click on any disconnected systems to open the details page
6. verify that the disconnected icon exists on the detail header as well and that the distance between the last seen date and the icon is smaller.  


Last seen info in the details page before:
![image](https://user-images.githubusercontent.com/59481011/181231190-ccf14f16-e7e1-44f2-812f-4a2d74d6bc29.png)


Last seen info in the details page after:
![image](https://user-images.githubusercontent.com/59481011/181230999-54624be1-be46-4d66-8d49-68b81ecd3bfa.png)
